packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -792,6 +695,159 @@ class VmServiceWrapper implements VmService {
     );
   }
 
+  // Mark: Overrides for [DdsExtension]. It would help with logical grouping to
+  // make these extension methods, but that makes testing more difficult due to
+  // mocking limitations for extension methods.
+  Stream<Event> get onExtensionEventWithHistory {
+    return _maybeReturnStreamWithHistory(
+      _vmService.onExtensionEventWithHistory,
+      fallbackStream: onExtensionEvent,
+    );
+  }
+
+  Stream<Event> get onLoggingEventWithHistory {
+    return _maybeReturnStreamWithHistory(
+      _vmService.onLoggingEventWithHistory,
+      fallbackStream: onLoggingEvent,
+    );
+  }
+
+  Stream<Event> get onStderrEventWithHistory {
+    return _maybeReturnStreamWithHistory(
+      _vmService.onStderrEventWithHistory,
+      fallbackStream: onStderrEvent,
+    );
+  }
+
+  Stream<Event> get onStdoutEventWithHistory {
+    return _maybeReturnStreamWithHistory(
+      _vmService.onStdoutEventWithHistory,
+      fallbackStream: onStdoutEvent,
+    );
+  }
+
+  Stream<Event> _maybeReturnStreamWithHistory(
+    Stream<Event> ddsStream, {
+    required Stream<Event> fallbackStream,
+  }) {
+    assert(_supportedProtocolsInitialized.isCompleted);
+    if (_ddsSupported) {
+      return ddsStream;
+    }
+    return fallbackStream;
+  }
+  // Mark: end overrides for the [DdsExtension].
+
+  // Mark: Overrides for [DartIOExtension]. It would help with logical grouping
```

Comment:
no functional changes from here down. Just moved all the dart io extension overrides to be grouped together.